openpype/hosts/houdini/plugins/publish/validate_review_colorspace.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.houdini.api.action import SelectROPAction
+from openpype.hosts.houdini.api.colorspace import get_default_display_view_colorspace  # noqa
+
+import os
+import hou
```

Comment:
Don't we normally do this kind of imports inside the `process` method? I see that the `from openpype.hosts.houdini.api.colorspace` imports it at the top, so in this instance it wont make a difference

Reply:
I'm not sure tbh. 
I can find this import for example in many places in Houdini plugins, 
sometimes it's written at the beginning of the file and sometimes it's inside methods! 
```python
import hou
```

So, should we agree that if a particular import is only used within a method, should it be imported only within that method?

Reply:
The rationale for it being within is that then it can be loaded outside of houdini, but unsure if we are actually following this or not, so don't bother for now.

Reply:
> The rationale for it being within is that then it can be loaded outside of houdini, but unsure if we are actually following this or not, so don't bother for now.

I'd say that use case only really applies for plugins that aren't within the hosts' addon? E.g. the one for Deadline I can see that making sense (even though there it's currently not done)

Reply:
It is also affecting performance of plugin discovery - even though it happens only one when modules are loaded and then it  is much faster. But remember pymel 😱

Reply:
I moved import statement inside their relevant functions.